src/ucp/core/ucp_context.c

```diff
@@ -1626,3 +1626,25 @@ ucp_memory_type_detect_mds(ucp_context_h context, void *address, size_t size)
     /* Memory type not detected by any memtype MD - assume it is host memory */
     return UCS_MEMORY_TYPE_HOST;
 }
+
+uint64_t ucp_context_tl_bitmap(ucp_context_h context, const char *dev_name)
+{
+    uint64_t        tl_bitmap;
+    ucp_rsc_index_t tl_idx;
+
+    if (dev_name == NULL) {
```

Comment:
pls rename this function to ucp_context_dev_tl_bitmap()
and no need to handle dev=NULL in this function. user which passes NULL should use context->tl_bitmap directly

Reply:
fixed in #4283